pkgs/development/ocaml-modules/tezos/protocol-008-PtEdo2Zk.nix

```diff
@@ -0,0 +1,23 @@
+{ lib
+, buildDunePackage
+, tezos-stdlib
+, tezos-protocol-compiler
+}:
+
+buildDunePackage {
+  pname = "tezos-protocol-008-PtEdo2Zk";
+  inherit (tezos-stdlib) version src useDune2 doCheck;
+
+  preBuild = ''
+    rm -rf vendors
+    sed -i.back -e s/-nostdlib//g src/proto_008_PtEdo2Zk/lib_protocol/dune.inc
```

Comment:
What about using `substituteInPlace`?

Reply:
Didn't know about this, just copied from the opam build